### PR TITLE
Cancel the invalid hyperlink

### PR DESCRIPTION
**Pull request by xichengliudui**
I clicked on an invalid hyperlink while using it and then canceled it

### CODE REVIEW

**Comment by DStorck** [comment]
lgtm

**Comment by neolit123** [comment]
/hold

@xichengliudui 
thanks for the PR.
please try adding  the `kubeadm ....http://` commands in "`" quotes instead. this should disable the links.

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ provide a file - a subset of the standard kubeconfig file. This file
 can be a local file or downloaded via an HTTPS URL. The forms are
 kubeadm join --discovery-token abcdef.1234567890abcdef 1.2.3.4:6443,
 kubeadm join --discovery-file path/to/file.conf, or kubeadm join
---discovery-file https://url/file.conf. Only one form can be used. If
+--discovery-file "https://url/file.conf". Only one form can be used. If
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md]: please add the following examples in code quotes - not the regular ' " ' quote but ' ` ' 

it should look like this:
> `kubeadm join --discovery-token abcdef.1234567890abcdef 1.2.3.4:6443`,
`kubeadm join --discovery-file path/to/file.conf`, or `kubeadm join
--discovery-file https://url/file.conf`. Only one form can be used. If....

**Pull request merged**